doc/sphinx-guides/source/api/search.rst

```diff
@@ -91,16 +103,26 @@ https://demo.dataverse.org/api/search?q=trees
                     "dataset_name": "Dataset One",
                     "dataset_id": "32",
                     "dataset_persistent_id": "doi:10.5072/FK2/XTT5BV",
-                    "dataset_citation":"Spruce, Sabrina, 2016, \"Spruce Goose\", http://dx.doi.org/10.5072/FK2/XTT5BV, Root Dataverse, V1"
+                    "dataset_citation":"Spruce, Sabrina, 2016, \"Spruce Goose\", http://dx.doi.org/10.5072/FK2/XTT5BV, Root Dataverse, V1",
+                    "publicationStatuses": [
+                        "Published"
+                    ],
+                    "releaseOrCreateDate": "2016-05-10T12:53:39Z"
                 },
                 {
                     "name":"Birds",
                     "type":"dataverse",
                     "url":"https://demo.dataverse.org/dataverse/birds",
-                    "image_url":"https://demo.dataverse.org/api/access/dvCardImage/2",
+                    "image_url":"data:image/png;base64,iVBORw0...",
                     "identifier":"birds",
                     "description":"A bird Dataverse collection with some trees",
-                    "published_at":"2016-05-10T12:57:27Z"
+                    "published_at":"2016-05-10T12:57:27Z",
+                    "publicationStatuses": [
+                        "Published"
+                    ],
+                    "affiliation": "Dataverse.org",
+                    "parentDataverseName": "Root",
+                    "parentDataverseIdentifier": "root"
                 },
                 {  
                     "name":"Darwin's Finches",
```

Comment:
Datasets can have `identifier_of_dataverse` but not `parentDataverseIdentifier`, right?

Reply:
Yes. parentDataverseIdentifier is only for Dataverses.